Makefile

```diff
@@ -11,7 +11,7 @@ check-lint: check-tabs
 		-type f \
 		-not -name \*.yaml \
 		-not -name \*.swp \
-		| xargs shellcheck && echo Success!
+		| xargs shellcheck --severity=warning && echo Success!
```

Comment:
From the commit message:
```
tests: Raise severity level for shellcheck

The `shellcheck` linter provides info-level messages for widely
used shell idioms (SC2015 for example).

These messages may be useful for future travelers, so instead of
suppressing them individually, raise the severity level to warning
in Makefile and subsequently CI.
```

Reply:
thanks